unit_tests/testLinearSolver/testLinearSolver.f90

```diff
@@ -3021,6 +3020,96 @@ SUBROUTINE testIterativeSolve_GMRES()
   DEALLOCATE(thisX)
   CALL thisLS%clear()
 
+  ! Banded matrix test
+  COMPONENT_TEST('BandedMatrixType')
+  ! initialize linear system
+  CALL pList%clear()
+  CALL pList%add('LinearSolverType->TPLType',LS_NATIVE)
+  CALL pList%add('LinearSolverType->solverMethod',GMRES)
+  CALL pList%add('LinearSolverType->MPI_Comm_ID',PE_COMM_SELF)
+  CALL pList%add('LinearSolverType->numberOMP',1_SNK)
+  CALL pList%add('LinearSolverType->timerName','testTimer')
+  CALL pList%add('LinearSolverType->matType',BANDED)
+  CALL pList%add('LinearSolverType->A->MatrixType->n',9_SNK)
+  CALL pList%add('LinearSolverType->A->MatrixType->m',9_SNK)
+  CALL pList%add('LinearSolverType->A->MatrixType->nnz',33_SNK)
+  CALL pList%add('LinearSolverType->x->VectorType->n',9_SNK)
+  CALL pList%add('LinearSolverType->b->VectorType->n',9_SNK)
```

Comment:
Is the _SNK necessary?

Reply:
These were copies from existing tests. Removing them changed nothing so I'll commit an updated unit test file with them all removed (seeing as they're pointless)

Reply:
These were copies from existing tests. Removing them changed nothing so I'll commit an updated unit test file with them all removed (seeing as they're pointless)